### PR TITLE
Clawdit

### PR DESCRIPTION
**Pull request by seth-shaw-unlv**
**GitHub Issue**: https://github.com/Islandora/documentation/issues/1267

# What does this Pull Request do?

Updates the conrtibuting.md to match the template and removes references to CLAW.

# How should this be tested?

Preview and review.

# Additional Notes:

There is still an open question about the README. It is currently structured as a tutorial. So, should we
1. revise it to be consistent with the template but leave the bulk of it as-is; OR 
2. move the existing README content to a new TUTORIAL.md file and link it from a new README matching the template?

The last remaining CLAW references are to https://github.com/Islandora-Devops/migrate_7x_claw which needs to be declawed itself.

# Interested parties
@manez, @Islandora/8-x-committers

### CODE REVIEW

**Comment by dannylamb** [comment]
I planned on making the README match the others and pushing the whole tutorial over into documentation.

**Comment by seth-shaw-unlv** [comment]
@manez & @dannylamb take a look and let me know what you think.

**Comment by seth-shaw-unlv** [comment]
Thanks, @manez!

File: README.md [hunk 1]
```diff
@@ -1,86 +1,29 @@
-# Migrating to Islandora 8 using CSVs
+# ![Mascot](https://user-images.githubusercontent.com/2371345/67424058-caac1a00-f5ab-11e9-99a2-c9360370391f.png) Migrating to Islandora 8 using CSVs
 
-## Table of Contents
+[![Build Status][1]](https://travis-ci.com/Islandora/migrate_islandora_csv)
```
Comment (manez) [inline README.md]: These aren't rendering properly. Looks like the references aren't there?

**Pull request merged**